.gitignore

```
@@ -1,4 +1,9 @@
 venv/
 .idea/
 
+lambda_function.zip
+climate_net_api/lambda_function.zip
+data_to_rds/lambda_function.zip
 config.py
+
+data_to_rds/__pycache__/
```

Similarly, there's no need to list individual directories for `__pycache__`; using `__pycache__/` will cover all `__pycache__` directories.

```suggestion
__pycache__/
```